packages/api-catalog-service/src/graph/namespace/typedef.graphql

```diff
@@ -0,0 +1,214 @@
+# Enums
+
+enum NamespaceSortType {
+  CREATED_ON
+  UPDATED_ON
+  NAME
+}
+
+enum NamespaceSortDir {
+  ASC
+  DESC
+}
+
+enum ApiCategory {
+  REST
+  GRAPHQL
+}
+
+# API Header
+
+type HeaderType {
+  id: ID
+  key: String
+}
+
+input HeaderInput {
+  key: String!
+  value: String!
+}
+
+input UpdateHeaderInput {
+  id: ID
+  key: String
+  value: String
+}
+
+# API Env Flag
+
+type ApiEnvironmentFlags {
+  isInternal: Boolean
+  isDepreciated: Boolean
+}
+
+input ApiEnvironmentFlagInput {
+  isInternal: Boolean!
+  isDepreciated: Boolean!
+}
+
+# API Namespaces
+type NamespaceType {
+  id: ID
+  name: String
+  slug: String
+  description: String
+  # tags: [String]
+  owners: [ApiOwnerType]
+  schemas: [ApiSchemaType]
+  createdOn: DateTime
+  createdBy: ApiUserType
+  updatedOn: DateTime
+  updatedBy: ApiUserType
+}
+
+input CreateNamespaceInput {
+  name: String!
+  description: String!
+  # tags: [String]
+  owners: [ApiOwnerInput]!
+  schemas: [CreateApiSchemaInput]!
+}
+
+input UpdateNamespaceInput {
+  name: String
+  description: String
+  # tags: [String]
```

Comment:
Why tags are commented if its not used we can remove it

Reply:
Removed